src/collections/configure/index.ts

```diff
@@ -311,5 +311,6 @@ export {
   tokenization,
   vectorDistances,
   configureVectorIndex as vectorIndex,
+  legacyVectors as vectorizer,
```

Comment:
This fixes another [breaking change](https://github.com/weaviate/typescript-client/pull/264/files?file-filters%5B%5D=.ts&show-viewed-files=true#diff-a3357c4e265a0992e8b0b8168ee30f635918ffdcfa03edb3db9c830b955d00ecL312-R314): in addition to maintaining `configure.vectorizer` we have to maintain `vectorizer` export.

Reply:
Good catch! Very true